src/migrations/20201019191554-add-business.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('business', {
+      id: {
+        type: Sequelize.UUID,
+        primaryKey: true,
+        defaultValue: Sequelize.UUIDV4
+      },
+      fantasyName: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      legalInformationId: {
```

Comment:
The FK should be in the LegalInformation table ("LegalInformation belongs to Business", "Business has 1 LegalInformation").